src/editDescription.js

```diff
@@ -0,0 +1,9 @@
+export default function editDescriptions(items) {
```

Comment:
- [] Please try to use the newer es6 syntax in situations where it is possible. It is recommended to use the newer arrow function syntax.
something like this would be great
```suggestion
export default (items) => {
```